app/assets/stylesheets/application/panel.scss

```diff
@@ -0,0 +1,16 @@
+.panel {
+  .actions {
+    form, button {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line